recipes/packages.rb

```diff
@@ -16,17 +16,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# include Php::Cookbook::ResourceHelpers
+
 include_recipe 'osl-selinux'
+Chef::DSL::Recipe.include Php::Cookbook::Helpers
 
-version = node['php']['version']
+version = Php::Cookbook::Helpers.php_version
+###version = php_version
```

Comment:
This should work now if you do what I suggested above.
```suggestion
version = php_version
```

Reply:
I think that worked, thank you!